### PR TITLE
v1.13: ci: use Ariane to trigger workflows

### PR DESCRIPTION
**Pull request by nbusseneau**
Please see the main PR at #26237 for full context and discussions.

### CODE REVIEW

**Comment by nbusseneau** [comment]
/ariane-test-backport-1.13

**Comment by nbusseneau** [comment]
/ariane-test-backport-1.13

**Comment by nbusseneau** [comment]
/ariane-test-backport-1.13

**Comment by nbusseneau** [comment]
/ariane-test-backport-1.13

**Comment by nbusseneau** [comment]
image
Removing temporary test commit.

File: .github/actions/gke/test-config-classic.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# List of test cases for conformance-gke with helm installation options
```
Comment (viktor-kurchenko) [inline .github/actions/gke/test-config-classic.yaml:1]: Copy paste? )

Comment (nbusseneau) [reply .github/actions/gke/test-config-classic.yaml:1]: @viktor-kurchenko That's just how the file is in `main`, it was probably a copy-paste issue there. If you want to fix it, let's fix in `main` :p

**Review by aanm — APPROVED** [review]

**Pull request merged**